guides/WRITING-DOCUMENTATION.md

```diff
@@ -15,176 +15,147 @@ For a component named "Foo", you would have the following:
 
-This is a required field, but if you want to create a subsection without a heading, you can enter `---`.
+**note:** If you want to create a subsection without a heading, add `---` after the opening `/*`.
+
+### Description
+
+The description should describe what pattern does or should be used for in plain language.
 
-### @flags
+### Flags
 
 To extend the default functionality of KSS, we've implemented support for custom flags. Flags should always come after the description.
 
 Supported flags:
 
 - **`@hide-example`** Hides the example and code snippet. Useful for rendering just the prop docs for React components.
 - **`@hide-markup`** Hides the code snippet.
-- **`@react-component [NAME]`** Displays the React component's example and `PropType` documentation.
-  - where `[NAME]` is a component's name relative to the source documentation file, or a path (without a file extension) relative to `packages`
+- **`@react-component [NAME]`** Displays the React `.jsx` component example, documentation and `Props`.
+    - **Example**: `@react-component Button` or `@react-component core/src/Components/Button/Button`.
+    - `[NAME]` is a component's name relative to the source documentation file, or a path (without a file extension) relative to `packages`. See [File naming](#file-naming) for more info.
 
-    **Example**: `@react-component Button` or `@react-component core/src/Components/Button/Button`.
-
-    See [File naming](#file-naming) above for more info.
-- **`@react-example [NAME]`** Displays an example using this React component.
+- **`@react-example [NAME]`** Displays example.jsx file using this React component. 
+    - **Example**: `@react-component Mask` displays `Mask.example.jsx`
+ 
 - **`@responsive`** Renders breakpoint toggles for the markup example.
-- **`@status [alpha|beta]`** Displays a status badge. Supported values: `alpha`, `beta`.
-- **`@uswds [URL]`** Marks the component as a US Web Design Standard component. Enter the URL so the documentation can link to the corresponding USWDS page.
-
-### Markup
+- **`@status [NAME]`** Displays a status badge. Supported values: `Draft`, `Work in progress`, `Ready`, `Deprecated`.
+- **`@uswds [URL]`** Marks the component as a US Web Design System component. Enter the URL so the documentation can link to the corresponding USWDS page.
 
-Markup examples can be written in plain HTML or [EJS](http://ejs.co/). The markup can be written inline or within a separate `.html` or `.ejs` file, relative to the CSS file:
+### Modifiers
 
-<details>
-<summary>Inline HTML example</summary>
-
-```
-Markup:
-<button class="ds-c-button {{modifier}}">Button label</button>
-```
-</details>
+If the item you are documenting has multiple states or styles depending on added classes or pseudo-classes, you should document them in the modifiers section. [More info on modifiers from the KSS documentation site](https://warpspire.com/kss/syntax/).
 
+### Markup
```

Comment:
In theory I support this change, but I'd keep the original in the docs as well until we have time (or the will?) to rewrite the way our own system looks. Walk our talk, etc.

Reply:
That makes sense - I'll add a follow on issue for that work and update the language until we are ready.